apis/v1alpha2/validation/gateway.go

```diff
@@ -91,3 +92,18 @@ func validateListenerHostname(listeners []gatewayv1a2.Listener, path *field.Path
 	}
 	return errs
 }
+
+// validateTLSCertificateRefs validates the certificateRefs
```

Comment:
Can you change this comment to explicitly say that the `certificateRefs` must be set and not empty in this scenario? The same for `v1beta1`.

Reply:
Updated it.